packages/definitions/dist/fhir/r4/search-parameters-medplum.json

```diff
@@ -1318,6 +1318,223 @@
         "type": "token",
         "expression": "AsyncJob.status"
       }
+    },
+    {
+      "fullUrl": "http://hl7.org/fhir/SearchParameter/Group-characteristic-reference",
+      "resource": {
+        "resourceType": "SearchParameter",
+        "id": "Group-characteristic-reference",
+        "extension": [
+          {
+            "url": "http://hl7.org/fhir/StructureDefinition/structuredefinition-standards-status",
+            "valueCode": "trial-use"
+          }
+        ],
+        "url": "http://hl7.org/fhir/SearchParameter/Group-characteristic-reference",
+        "version": "5.0.0",
+        "name": "characteristic-reference",
+        "status": "draft",
+        "experimental": false,
+        "date": "2023-03-26T15:21:02+11:00",
+        "publisher": "Health Level Seven International (FHIR Infrastructure)",
+        "contact": [
+          {
+            "telecom": [
+              {
+                "system": "url",
+                "value": "http://hl7.org/fhir"
+              }
+            ]
+          },
+          {
+            "telecom": [
+              {
+                "system": "url",
+                "value": "http://www.hl7.org/Special/committees/fiwg/index.cfm"
+              }
+            ]
+          }
+        ],
+        "description": "An entity referenced in a characteristic",
+        "jurisdiction": [
+          {
+            "coding": [
+              {
+                "system": "http://unstats.un.org/unsd/methods/m49/m49.htm",
+                "code": "001",
+                "display": "World"
+              }
+            ]
+          }
+        ],
+        "code": "characteristic-reference",
+        "base": [
+          "Group"
+        ],
+        "type": "reference",
+        "expression": "(Group.characteristic.value.ofType(Reference))",
+        "processingMode": "normal",
```

Comment:
`processingMode` doesn't exist in R4, think if we remove that key it should work:
https://hl7.org/fhir/R4/searchparameter.html